docs/HOWTO-use-dynamic-text-script.md

```diff
@@ -0,0 +1,50 @@
+# Dynamic Text Script
```

Comment:
At first, I underestimated how much needed to get done because I thought this functionality already existed.  Within that assumption, I created this documentation to help someone who is an administrator "observe" their text content.

I was quickly corrected that the sample text needed to be available to the administrator from the campaign tab.

That began my hunting to find how I get all of the data points to render the text message the same way it's rendered for a Texter.